Model/Billing/Agreement.php

```diff
@@ -190,7 +193,10 @@ public function setCcBillingAgreement($contractDetail, $storeOneClick, $storeId)
         }
         // Billing agreement is CC
 
-        $ccType = $contractDetail['paymentMethod'];
+        $ccType = $variant = $contractDetail['paymentMethod'];
```

Comment:
why do we need to vars for the same value?

Reply:
ccType will be replaced in line 203 and will be the Magento card type, we still need the adyen type to save in 252